### PR TITLE
#480 Content Checking script for new language genearion of ftm!

### PR DESCRIPTION
**Pull request by amitsinghsutara**

### CODE REVIEW

File: Automation Scripts/content_checking_script.py [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+        return
+
+    for folder in folders:
+        if folder["id"] == desired_folder_id:
+            # You've reached the desired folder, now you can list its contents or perform any desired actions.
+            list_contents_of_folder(drive_id, folder["id"],unique_prompt_texts,lang,wav_unique_prompt_texts,m4a_prompt_text, depth + 1)
+           
+                        
+        else:
+            if folder["id"]=="1241p6yE0at78OZVdTewJSWS6XdjPjx93" or folder["id"]=="1lsbOukZZTFGJI8tTMRGDTi1QpuiAGJaR" or folder["id"]=="1dVw3lRUcP0vHP8qGhGqeHmG0Elflx2HY" or folder["id"]=="1xTln0bFAGJe3hhWu_KdpdQb7gQcSAktG":
```
Comment (Rajesh1041) [inline Automation Scripts/content_checking_script.py:314]: Can we shorten these kinds of long lines to increase readability?

File: Automation Scripts/content_checking_script.py [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+    return missing_audios_on_drive
+
+
+
+
+
+
+
+
+
```
Comment (Rajesh1041) [inline Automation Scripts/content_checking_script.py:327]: Can we lessen the line breaks?

File: Automation Scripts/content_checking_script.py [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+    
+    
+    response = requests.get(url)
+
+# Check if the request was successful (status code 200)
+    if response.status_code == 200:
+    # Parse the JSON content
+        json_data = response.json()
+        
+
```
Comment (Rajesh1041) [inline Automation Scripts/content_checking_script.py:388]: Uneven spaces between lines?

File: Automation Scripts/content_checking_script.py [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+    print(">>>>>>>>>>>>>",filename)
+    return filename.lower()
+
+
+
+
+
+
+def list_contents_and_download(drive_id, folder_id,lang,wav_unique_prompt_texts,m4a_prompt_text,unique_prompt_texts, depth=0):
+    # Query for contents in the selected folder
```
Comment (Rajesh1041) [inline Automation Scripts/content_checking_script.py:215]: Comment should be right above the line of code.

File: Automation Scripts/content_checking_script.py [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+        file_name=get_correct_file_name(file_name)
+        file_name=unicodedata.normalize('NFKD', file_name)
+        print(str(count)+"___"+file_name+" "+"checking in"+" "+fileName)   
+                
+                
+        
+        if file_name in wav_unique_prompt_texts or file_name in unique_prompt_texts  or file_name in m4a_prompt_text:
+            if file_name not in present_audios_on_drive:
+                name=file_name.lower()
+
```
Comment (Rajesh1041) [inline Automation Scripts/content_checking_script.py:176]: Uneven spacing between lines!

File: Automation Scripts/content_checking_script.py [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+                        
+def find_prompt_audios(data, prompt_audio_urls=None):
+    feedback_words = ['amazing.mp3', 'fantastic.mp3', 'great1.mp3', 'amazing.mp3']
+    if prompt_audio_urls is None:
+        prompt_audio_urls = set()
+        
+    for feedback in feedback_words:
+        prompt_audio_urls.add(feedback)
+    try:
+
```
Comment (Rajesh1041) [inline Automation Scripts/content_checking_script.py:79]: Uneven spaces!

File: Automation Scripts/content_checking_script.py
Comment (Rajesh1041) [inline Automation Scripts/content_checking_script.py:1]: Same kinds of changes might require at multiple places, please correct before submitting again!

**Comment by amitsinghsutara** [comment]
Thanks @Rajesh1041 ,
For your feedback,
Sorry for the delay, I was on leave!
Will update it soon today as per above mentioned instructions!
Best Regards
Amit

**Pull request merged**